ledger/accountdb.go

```diff
@@ -2977,6 +3021,42 @@ func onlineAccountsExpirations(tx *sql.Tx, maxBalLookback uint64) (result []onli
 	return result, nil
 }
 
+func onlineAccountsAll(tx *sql.Tx) ([]persistedOnlineAccountData, error) {
+	rows, err := tx.Query("SELECT rowid, address, updround, data FROM onlineaccounts ORDER BY address, updround ASC")
```

Comment:
I think we should query with a limit here.

Reply:
Is there a good way to make sure that wherever the limit is, we dont return acct data for only some of the rounds for a particular account

Reply:
how about
```
SELECT ... FROM onlineaccounts WHERE address IN (SELECT address FROM onlineaccounts LIMIT 99) ORDER BY address, updround ASC
```

Reply:
oh wait

Reply:
I think you could use the same query but read rows until there are too many accounts.

Reply:
done with maxAccount paramenter